spyder/config/main.py

```diff
@@ -516,6 +516,10 @@
               'show_installation_dialog': True,
               'show_onboarding': True,
              }),
+            ('find_in_files',
+             {
+              'max_results': 1000,
+              }),

```

Comment:
`find_in_files` already has its own section, so please add this entry there.

Also, please change the `more_options` of `find_in_files` to False, to hide the exclusion options by default from our config.

Reply:
Fixed (I actually had added the option in the wrong place)